packages/frontend-asset-loader/src/utils/logger.ts

```diff
@@ -0,0 +1,11 @@
+import { pino } from "pino";
+
+export const logger = pino({
```

Comment:
By default this Pino instance will get initialised when the application starts-up, even if it's not used. This could also mean that it throws an error if the value of `process.env.LOG_LEVEL` is not recognised.

Could this be changed so that the instance is only initialised when it's going to be used? Also as part of that we'd want to make sure it can only be initialised once—essentially I think we want a Singleton pattern here, where multiple calls to get the Pino logger would return the same instance.